src/Tasks/GenerateResource.cs

```diff
@@ -42,6 +42,7 @@
 using Microsoft.Build.Utilities;
 #if FEATURE_RESXREADER_LIVEDESERIALIZATION
 using Microsoft.Win32;
+using System.Windows.Forms;
```

Comment:
Oh hack - wraping my head around how the code adjustments lead to need for `Froms` :hushed:
But it's conditioned for Full FW - so probably all good